### PR TITLE
Use camel case in messages instead of underscore.  Historically, that…

### PR DESCRIPTION
**Pull request by chitang**
…’s what we have been doing.

### CODE REVIEW

**Comment by catstavi** [comment]
There's a couple other classes that use the underscore strategy: `ComputationStrategySpec`, and `SecondariesOfDatasetResult`-- is there a reason those are ok to use the underscore strategy but these aren't?

**Comment by chitang** [comment]
ComputationStrategySpec is SODA2.  Eurybates messages use camel case.

File: coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/messaging/Message.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ sealed abstract class Message
 // We aren't using this message right now, but I have hopes to use it for
 // webhooks or other things that want to know about when NBE replication
 // is complete. So going to leave this here for now.
-@JsonKeyStrategy(Strategy.Underscore)
+@JsonKeyStrategy(Strategy.Identity)
 case class StoreReplicationComplete(viewUid: ViewUid,
```
Comment (aerust) [inline coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/messaging/Message.scala]: I might just omit the JsonKeyStrategy, Identity is the default

**Comment by aerust** [comment]
Can you include in the commit message that it is what we do historically for Eurybates just to be clear. Also nice to keep first line of commit messages under 60 characters.

**Comment by aerust** [comment]
looks great :ship:

**Pull request merged**